src/wx/xrc/GeneralConfig.xrc

```diff
@@ -111,31 +111,31 @@
               <object class="sizeritem">
                 <object class="wxChoice" name="ThrottleSel">
                   <content>
-                    <item>Full Speed/No Limit</item>
-                    <item>25%</item>
-                    <item>50%</item>
-                    <item>75%</item>
-                    <item>100%</item>
-                    <item>125%</item>
-                    <item>150%</item>
-                    <item>175%</item>
-                    <item>200%</item>
-                    <item>225%</item>
-                    <item>250%</item>
-                    <item>275%</item>
-                    <item>300%</item>
-                    <item>325%</item>
-                    <item>350%</item>
-                    <item>375%</item>
-                    <item>400%</item>
-                    <item>425%</item>
-                    <item>450%</item>
-                    <item>475%</item>
-                    <item>500%</item>
-                    <item>525%</item>
-                    <item>550%</item>
-                    <item>575%</item>
-                    <item>600%</item>
+                    <item>Unlimited</item>
```

Comment:
I am having some issues with how this is being displayed on `GTK3, wxWidgets 3.0.1`.
![Screenshot_2020-05-12_12-27-50](https://user-images.githubusercontent.com/11354751/81713295-0d501980-944c-11ea-8854-9636af470c2d.png). Removing this line fixes this.

Reply:
Will take a look.